composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
+        "php": ">=5.6",
```

Comment:
I like adding `^5.6 || ^7.0` instead because we do not know if we support php 8 and we do not support php 6. 
Better to be specific.